static/js/map.js

```diff
@@ -7117,11 +7209,9 @@ $(function () {
         var options = {
             'duration': 500
         }
-        var wrapper = $('#quests-filter-wrapper')
         var rocketWrapper = $('#rocket-wrapper')
         if (this.checked) {
             lastpokestops = false
-            wrapper.hide(options)
```

Comment:
With this change the quest filter doesnt get hidden when you toggle rocket only.

Reply:
you will see why in my next commits. "wrapper" is not defined in this function it worked because it is defined in the function above.